docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.1
+    environment:
+      CALYPSO_ENV: 'development'
```

Comment:
let's also put in `NODE_ENV: 'development'` unless that's redundant. I don't think it is